pkgs/applications/audio/open-stage-control/default.nix

```diff
@@ -1,29 +1,26 @@
-{ pkgs, stdenv, lib, fetchFromGitHub, makeBinaryWrapper, makeDesktopItem, copyDesktopItems, nodejs, electron, python3, ... }:
+{ lib, buildNpmPackage, fetchFromGitHub, makeBinaryWrapper, makeDesktopItem, copyDesktopItems, electron, python3, ... }:
```

Comment:
Do we need the ... here?

Reply:
Oh yeah we don't, thanks! I'll remove it when I'm next at my computer

Reply:
Done!